docs/accuracy_aware_model_training/EarlyExitTrainig.md

```diff
@@ -0,0 +1,35 @@
+# EarlyExit training loop in NNCF
+
+Early-Exit training loop aims to get the compressed model with a desired accuracy criteria as earlier as possible. After NNCF initialization algorithms applied and then after each trainig epoch the loop measures accuracy of the compressed model and if it is reach the user-defined criteria exits and returns a compressed model.
+This pipeline is simple but effective. For many models it reduces training time till just an initialization step. 
+
+This training loop supports any combination of NNCF compression algorithms.
+
```

Comment:
Please add this note: `Note: since the EarlyExit training does not control any compression parameter the specified accuracy criterium cannot be satisfied in some cases`

Reply:
Done